src/components/BoxAnimated/BoxAnimated.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect, useCallback, useRef, useMemo } from "react";
```

Comment:
i think this is the code for the scroll down animation? perhaps a comment for those, like me!, who are abit behind in skills might be helpful :-) I LOVE this animation, it is clean and not over the top, flows really well, great job!